examples/tests/valid/external_ts.extern.d.ts

```diff
@@ -51,7 +51,7 @@ export interface MetadataEntry {
 export class Node {
   /** Add an ordering dependency on another construct.
   An `IDependable` */
-  readonly addDependency: (deps?: ((readonly (IDependable)[])) | undefined) => void;
+  readonly addDependency: (deps: (readonly (IDependable)[])) => void;
```

Comment:
Should it generate something like this?
```suggestion
  readonly addDependency: (...deps: (readonly (IDependable)[])) => void;
```

The compiler's `FunctionParameter` struct has a `variadic` boolean on it which can be used for this I think

Reply:
Ooo love latent bugs

Reply:
Created https://github.com/winglang/wing/issues/6512 since it's an existing issue.

The downside of boolean flags instead of an enum to force handling of different cases

Reply:
Thanks @MarkMcCulloh